src/helper/certificate.ts

```diff
@@ -133,17 +133,24 @@ export async function generateCertificate(
     "font.fnt"
   );
   console.log(fontPath);
+  const alignment = event.certificate?.alignment;
+
   imgObject = await imgObject.print(
     await Jimp.loadFont(fontPath),
     certParams!.x,
     certParams!.y,
     {
       text: name,
       alignmentX:
-        event.slug === "mozofest2022-parti"
-          ? Jimp.HORIZONTAL_ALIGN_CENTER
-          : Jimp.HORIZONTAL_ALIGN_LEFT,
-      alignmentY: Jimp.VERTICAL_ALIGN_MIDDLE,
+        
+      alignment === "right"
+      ? Jimp.HORIZONTAL_ALIGN_RIGHT
+      : alignment === "left"
+         ? Jimp.HORIZONTAL_ALIGN_LEFT
+         : alignment === "center"
+           ? Jimp.HORIZONTAL_ALIGN_CENTER
+           : Jimp.HORIZONTAL_ALIGN_LEFT,
```

Comment:
This defaults to left. Why? If it is right, do right else check if it is left, if true, then left or center. Why are there so many conditions? it will only increase the complexity.